### DISCUSSION TITLE
How can I add different paddings depending on screen size?

### DISCUSSION THREAD

**Discussion by mucahidyazar** [resend/react-email#1297] [Q&A] [answered]
I just wonder if don't use tailwind how can I have a responsive style with react-email?

For ex: I want 20px padding-top in desktop, 8px in mobile. How can I archive that by using react-email or style attribute?

**Comment by mucahidyazar** [comment]
And I think the documentation is missing this. There should be a styling section.

**Reply by jbouhier** [reply]
I agree. More generally a section about the different ways to do RWD / media queries / CSS. Can I help with this @gabrielmfern?

**Comment by gabrielmfern** [comment, marked as answer]
I am not sure if we need this, this is more of a learn-how-to-write email HTML than a learn-React-Email thing.

Answering your question though, you can just add a `<style>` tag on your Head and there use media queries. It is the same
as what Tailwind does right now. Just be careful with escaping characters.
